src/types/apy.ts

```diff
@@ -0,0 +1,78 @@
+enum SecretQueryOptions {
```

Comment:
I'd be fine moving this out of the types and to the service layer so that it is more easily viewable as to what the endpoints are. What do you think?